bag/datasets/bag/batch_gob.py

```diff
@@ -862,7 +1351,12 @@ def process_row(self, r):
         eigendomsverhouding_id = get_code_for_omschrijving(r['eigendomsverhouding'], models.Eigendomsverhouding,
                                                            self.eigendomsverhoudingen)
         gebruik_id = get_code_for_omschrijving(r['is:WOZ.WOB.soortObject'], models.Gebruik, self.gebruik)
-        toegang_id = get_code_for_omschrijving(r['toegang'], models.Toegang, self.toegang)
+
+        # TODO : In GOB there can be multiple toegangen in the toegang field.
+        # For now we use the first entry. We have to determine how to deal with this later
```

Comment:
Is daar een issue voor gemaakt?

Reply:
Nee, in principle was Karin OK dat we dat nu zo doen. Ik denk dat dat getriggerd moet worden vanuit BI dat dit echt ondersteund gaat worden. Het staat er alleen bij ter informatie

Reply:
(Misschien dan de `TODO` weg?)

Reply:
Ik zal hem weghalen